app/_assets/stylesheets/features-table.less

```
@@ -0,0 +1,68 @@
+.features-table {
+  width: fit-content;
+
+  .feature-cta {
+    display: inline-block;
+    padding: .5em;
+    border-radius: 5px; 
+    border: 1px solid @blue;
+    color: @blue;
+    width: 11em;
+    margin-top: 12px;
+
+    &:hover {
+      background: @blue;
+      color: white;
+      text-decoration: none;
+    }
+  }
+
+  // Undo old styling
+  tr:nth-child(2n), tr:nth-child(2n+1) {
+    background-color: transparent;
+    border: 0;
+  }
+
+  .product-name {
+    width: 12em;
+  }
+
+  tbody tr:first-child {
+    .header-row {
+      margin-top: 0px;
+    }
+  }
+
+  .header-row {
+    font-size: 18px;
+    font-weight: bold;
+    color: @blue;
+
+    margin-top: 24px;
+    display: block;
+
+    span {
+      border-bottom: dashed #a6c6ff 1px;
+      padding-bottom: 5px;
+    }
+  }
+
+  th {
+    vertical-align: top;
+    img {
+      height: 65px;
```

```suggestion
      height: 65px;
      border-radius: 5px
```
To match slightly more, since that pricing page is all curved corners.